docs/en/setup/backend/dynamic-config.md

```diff
@@ -31,9 +31,31 @@ configuration:
     #clusterName: "default" # the name of current cluster, set the name if you want to upstream system known.  
 ```
 
+## Nacos DCS
+
+[Nacos](https://github.com/alibaba/nacos) is also supported in DCS, to use it, please configure as follows:
+
+```yaml
+configuration:
+  nacos:
+    # Nacos Server Host
+    serverAddr: 127.0.0.1
+    # Nacos Server Port
+    port: 8848
+    # Nacos Configuration Group
+    group: 'skywalking'
+    # Dynamically configured keys
+    data-ids:
+      - receiver-trace.default.slowDBAccessThreshold
```

Comment:
In Nacos side, this Key is not needed, right?

Reply:
It needs, Nacos needs to know which key(dataId) we are interested in

Reply:
OK